spec/active_record/typed_store_spec.rb

```diff
@@ -692,6 +692,34 @@
 
   end
 
+  describe 'with prefix' do
+
+    it 'defines accessors' do
+      expect(model).to respond_to :preferences_language
+      expect(model).to respond_to :preferences_language=
```

Comment:
Yep, I need to look into how the attribute definition works as well. The values does not seem to be set correctly. I'll ping back once I've figured it out.